opentutor_classifier/tests/test_lr_feature_gen.py

```diff
@@ -230,6 +231,7 @@ def test_generates_features_when_env_train_quality_1(
                 "featureRegexAggregateDisabled",
                 "archetype_good",
                 "archetype_bad",
+                "TRAIN_QUALITY",
```

Comment:
don't do this, use a property name that indicates what the feature is as already exists with `featureRegexAggregateDisabled`, so this should be like `featureDbScanClustersEnabled` or something similar